### PR TITLE
#166265853 Implement view and delete books

### PR DESCRIPTION
**Pull request by chuxmykel**
#### What does this PR do?
This PR adds the 'view specific book' and 'delete book' functionality to the application.

#### Description of task to be completed
- user should be able to fetch all books
- user should be able to fetch a specific book
- user should be able to delete a specific book
- work on password reset validation

#### How should this be manually tested
- send a `GET` request to `/books/<id>`.
- send a `DELETE` request to `/books/<id>`.

#### Any background context you want to provide
- None

#### Relevant Pivotal Tracker stories
- [#166265853](https://www.pivotaltracker.com/n/projects/2351121/stories/166265853)

#### Screenshots
Screenshot from 2019-06-07 08-27-46
Screenshot from 2019-06-07 08-26-55
Screenshot from 2019-06-07 07-45-55
Screenshot from 2019-06-07 07-32-01

### CODE REVIEW

**Review by cvjude — DISMISSED** [review]

File: .env.sample [hunk 1]
```diff
@@ -1,4 +1,4 @@
-PORT = an integer representing the port through which the server will be accessed e.g. 3000
+ORT = an integer representing the port through which the server will be accessed e.g. 3000
```
Comment (emp-daisy) [inline .env.sample]: Hey! What's the change here?

**Comment by Hadeneekeh** [comment]
Hello Chux, well done. Since password reset validation is entirely different from viewing and deleting a book, I think it should not be included on this branch and subsequently on this PR.

**Comment by emp-daisy** [comment]
I asked him to include it there @Hadeneekeh

File: .env.sample (hunk 1 above)
Comment (Hadeneekeh) [reply .env.sample]: Okay

**Review by encodedBicoding — DISMISSED** [review]
Fix change in .env.sample file

**Comment by Hadeneekeh** [comment]
> I asked him to include it there @Hadeneekeh

Okay

**Review by ocranbillions — DISMISSED** [review]

File: .env.sample (hunk 1 above)
Comment (chuxmykel) [reply .env.sample]: Oh Boy. Error. Sorry. I'll fix that now

**Review by emp-daisy — APPROVED** [review]

**Review by Hadeneekeh — APPROVED** [review]

**Pull request merged**